appserver/packager/glassfish-common/pom.xml

```diff
@@ -177,9 +177,9 @@
             <artifactId>jakarta.enterprise.concurrent-api</artifactId>
        </dependency>
        <dependency>
-           <groupId>org.glassfish</groupId>
-           <artifactId>jakarta.enterprise.concurrent</artifactId>
-       </dependency> 
+            <groupId>org.glassfish.concurro</groupId>
+            <artifactId>concurro</artifactId>
+        </dependency>
```

Comment:
Still outstanding - you're adding 5 spaces here rather than 4.
I'd prefer if we fix the formatting of the other misaligned dependencies rather than compounding it.